examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/README.md

```diff
@@ -73,12 +73,13 @@ python run_clm_no_trainer.py \
 python run_clm_no_trainer.py \
     --model EleutherAI/gpt-j-6B \
     --accuracy \
+    --approach weight_only \
     --batch_size 112 \
     --tasks "lambada_openai"\
     --int8 \
-    --ipex \
     --output_dir "saved_results"  # load int8 model
 # to validate FP32 model, please remove "--int8" and "--output_dir".
+# to validate the int8 model quantized with ipex, please include "--ipex".
 ```
```

Comment:
please add one more comments, `# to validate int8 model generated by SmoothQuant, please remove "--approach weight_only". `

Reply:
Nice catch, added in [da1c6df](https://github.com/intel/neural-compressor/pull/1529/commits/da1c6df6e9b8dfc9e71baac56e19cb38b966fd9e).